### ISSUE TITLE
Default (package) access methods are not correctly handled

### ISSUE THREAD

**Issue by activeeon-bot** [ow2-proactive/programming#929]
<a href="https://jira.activeeon.com/browse/PROACTIVE-1053" title="PROACTIVE-1053">Original issue</a> created by <a href="mailto:christian.delbe_AT_activeeon.com">Christian Delbé</a> on 17, Jun 2011 at 16:25 PM - PROACTIVE-1053

<hr />

If an activated class defines a default (package) access method (i.e. no modifier), this method is overloaded in the stub BUT is not callable on the stub.
This is due to the fact that polymorphism does not apply if the overloaded method is not in the same package than the package access method. Indeed, stubs are generated in their own pa.stub... package.

Ex :
package p1
public class A {
 int m() {
   return 1;
}
}

package p2
public class B extends A {
 int m() { // could be public or not
   return 2;
}
}

A a = new B();
sysout(a.m());

==> 1

**Comment by activeeon-bot** [comment]
<a href="https://jira.activeeon.com/browse/PROACTIVE-1053?focusedCommentId=17925">Original comment</a> posted by <a href="mailto:brian.amedro_AT_activeeon.com">Brian Amedro</a> on 07, Aug 2014 at 08:39 AM

<hr />

Nobody is blocked on this bug

**Comment by activeeon-bot** [comment]
<a href="https://jira.activeeon.com/browse/PROACTIVE-1053?focusedCommentId=18634">Original comment</a> posted by <a href="mailto:youri.bonnaffe_AT_activeeon.com">Youri Bonnaffe</a> on 12, Dec 2014 at 13:07 PM

<hr />

Marking old issues as won't fix, either because we don't support it anymore, or it is needed or it has been there for too long.

Feel free to reopen if needed.
